src/components/ResultsList.vue

```diff
@@ -28,38 +32,43 @@ export default {
   data() {
     return {
       noResults: false,
-      sorted: false
-    }
+      noEmail: false,
+      sorted: false,
+    };
   },
   methods: {
     load() {
-      if (!this.$store.state.searchResults.length) {
-        this.noResults = true
+      if (!this.$store.state.email) {
+        this.noEmail = true;
+      } else if (!this.$store.state.searchResults.length) {
+        this.noResults = true;
       }
     },
     sortByRating() {
       this.$store.state.searchResults.sort((parkA, parkB) => {
-        return parkB.rating - parkA.rating
-      })
-      this.sorted = true
-    }
+        return parkB.rating - parkA.rating;
+      });
+      this.sorted = true;
+    },
   },
   mounted() {
-    setTimeout(() => this.load(), 6000)
+    setTimeout(() => this.load(), 4000);
   },
   components: { ResultsListItem, FindPark, LoadingIcon },
   computed: {
     message() {
```

Comment:
How does the loading message still show if the user doesn't have an email stored in Vuex?  From what I understand, `this.noEmail` should be true if the user visits the results too early, and shouldn't reach the second `else if` on line 65 that shows `Loading parks...`.  Am I misunderstanding, or are you referring to something else in your PR?

Reply:
Oh, never mind! I just remembered the `setTimeout`...